Core/src/Detector/VolumeStructureBuilder.cpp

```diff
@@ -175,8 +180,14 @@ Acts::Experimental::VolumeStructureBuilder::construct(
     default:
       break;
   }
+
+  auto fTransform = m_cfg.transform * eTransform;
```

Comment:
Suggest using the explicit type here for force evaluation.

Reply:
Correct.

Reply:
Done.